### DISCUSSION TITLE
Cherry-picking css components

### DISCUSSION THREAD

**Discussion by thor9n** [buefy/buefy#3544] [Q&A]
Hey!

The documentated way to include css is to `import 'buefy/dist/buefy.css`. I can live with importing unused components, but I cannot live with the css setting size, color and other subjective properties.

Is there any way to cherry-pick the css modules and only include css needed to objectively display a component in a functional way; e.g. an collapse/accordion sliding animation but no other styling?

Thanks in advance ✌🏻

**Comment by zizlfz** [comment]
Have you tried creating your own `buefy.scss` then import only what you need?
You can look at the source here: https://github.com/buefy/buefy/blob/dev/src/scss/buefy.scss
